client/js/components/user/DpUserFormFields.vue

```diff
@@ -71,25 +71,25 @@
       </label>
       <dp-multiselect
         v-if="hasPermission('area_organisations')"
-        :placeholder="Translator.trans('search.three.signs')"
-        :value="currentUserOrga"
-        :searchable="true"
+        :id="userId + ':organisationId'"
+        :close-on-select="true"
```

Comment:
Where is `closeOnSelect` used?

Reply:
It is used in the vue multiselect. I added a prop in DpMultiselect (default true). [24fc29a](https://github.com/demos-europe/demosplan-core/pull/1393/commits/24fc29a9bccda5c2b3a08969e327419eb9991a3f)